sdk/core/azure-core/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ Azure-core is supported on Python 3.7 or later. For more details, please read ou
 
 ### Features Added
 
+- Added Pyodide-compatible transport.
```

Comment:
We would have to add a strong disclaimer about this being experimental. I would probably even add a new heading "Experimental features".

Reply:
@johanste Just to clarify, we want to use "experimental" or "provisional"?

Reply:
used experimental